CHANGES.rst

```diff
@@ -0,0 +1,14 @@
+Changelog
+=========
+
+Unreleased
```

Comment:
We can do a release as 0.3.0. I set it up so that Travis will deploy to pypi when a PR is merged into master.